packages/atlas/src/components/_ypp/SelectChannelDialog/SelectChannelDialog.styles.ts

```diff
@@ -0,0 +1,5 @@
+import styled from '@emotion/styled'
+
+export const ListItemsWrapper = styled.div`
+  margin: 0 calc(-1 * var(--local-size-dialog-padding));
```

Comment:
Unfortunately, the padding is not correct here. It should be 24px
Yours: 
![image](https://user-images.githubusercontent.com/51168865/190597242-fda7a4e1-2ee0-4984-9fde-4de1826277a1.png)
Design:
![image](https://user-images.githubusercontent.com/51168865/190597047-16b095d5-536d-497c-bc9e-5489c66df0c4.png)
![image](https://user-images.githubusercontent.com/51168865/190597109-66355923-96d8-4c4a-9b86-402b31be72e7.png)